### PR TITLE
Bug hunting/log events

### PR DESCRIPTION
**Pull request by thallesfreitaszup**
## Issue Description
Events emitted by meta controller was not being handled by butler

## Solution

Upgrade metacontroller version, not require namespace to handle events

### CODE REVIEW

**Review by fandrade-zup — DISMISSED** [review]
LGTM

**Review by lucasbfernandes — DISMISSED** [review]

File: butler/src/app/v2/operator/logs-aggregator/kubernetes-events-aggregator.ts [hunk 1]
```diff
@@ -164,8 +154,58 @@ export class EventsLogsAggregator {
...
+      this.consoleLoggerService.log(`Event created at ${event.timestamp} is older then ${since}. Discarding event...`)
+      return
+    }
+    return event
+  }
+
+  private getDeploymentIdLabel(resource: k8s.KubernetesObject) {
+    const deploymentIdResource = resource.metadata?.labels?.[AppConstants.DEPLOYMENT_ID_LABEL]
+    if (!deploymentIdResource) {
+      if (resource.kind === AppConstants.CHARLES_CUSTOM_RESOURCE_DEPLOYMENT_KIND) {
```
Comment (leandrorodrigueszup) [inline butler/src/app/v2/operator/logs-aggregator/kubernetes-events-aggregator.ts:200]: I think would be nice put this in method, to expose this rule.

Comment (thallesfreitaszup) [reply butler/src/app/v2/operator/logs-aggregator/kubernetes-events-aggregator.ts]: done!

**Review by lucasbfernandes — APPROVED** [review]

File: butler/src/app/v2/operator/logs-aggregator/kubernetes-events-aggregator.ts [hunk 2]
```diff
@@ -164,9 +154,64 @@ export class EventsLogsAggregator {
...
+      return
+    }
+
+    if (involvedObject.namespace && involvedObject.namespace.startsWith(EventsLogsAggregator.KUBE_SYSTEM_NS_PREFIX)) {
+      this.consoleLoggerService.log(`${involvedObject.namespace} is a kubernetes system namespace. Discarding event...`, involvedObject)
+      return
+    }
+    const event = new Event(coreEvent)
+    if (this.isEventOlderThan(event, since)) {
+      this.consoleLoggerService.log(`Event created at ${event.timestamp} is older then ${since}. Discarding event...`)
```
Comment (eliasalveszup) [inline butler/src/app/v2/operator/logs-aggregator/kubernetes-events-aggregator.ts:191]: The correct is "older THAN"

Comment (thallesfreitaszup) [reply butler/src/app/v2/operator/logs-aggregator/kubernetes-events-aggregator.ts]: done!

**Review by leandrorodrigueszup — APPROVED** [review]

**Pull request merged**